tests/functional/ingestion/MongoQueueProcessor.js

```diff
@@ -127,7 +127,8 @@ class MongoClientMock {
     deleteObject(bucket, key, params, log, cb) {
         assert.strictEqual(bucket, BUCKET);
         assert([KEY, `${KEY}${VID_SEP}${VERSION_ID}`].includes(key));
-        this._deleted.push(key);
+        const versionId = params && params.versionId;
+        this._deleted.push({ key, versionId });
```

Comment:
This pushes the key-value pair `versionId: undefined` in all cases. Is this expected, or do we want to avoid pushing the key if we have no versionId?

Reply:
that's expected, we want to push both cases, to see if the params passed to the delete function are good